PolyEngine/Engine/Res/Shaders/transparent.vert.glsl

```diff
@@ -0,0 +1,41 @@
+#version 430 core
+layout(location = 0) in vec4 aPosition;
+layout(location = 1) in	vec2 aUV;
+layout(location = 2) in	vec3 aNormal;
```

Comment:
Codding style, strange space in attribute declaration.

Reply:
Done